.github/ISSUE_TEMPLATE/onborading.md

```diff
@@ -0,0 +1,18 @@
+---
+name: onBoarding projects
+title: "[onBoarding]: "
+labels: [onBoarding]
+---
+
+## Vault information
+
+In order to onboarding your project on our platform, you need to provide the correct crowdloan information as following:
+
+- [] ParaId: <!--This is the para ID of your project on relaychain. Make sure it is the right one. e.g. `2085`-->
+- [] Lease period: <!-- check the slot duration of Auction schedule, check it from https://kusama.network/auctions/ and https://polkadot.network/auctions/ -->
+- [] Cap: <!-- what's your crowdloan cap balance on relaychain when creating crowdloan? -->
+- [] EndBlock <!-- what's your endblock on relaychain when creating crowdloan? -->
+
+## Project information
+
+In order to show the correct information (name, logo, rewards rules etc), you need to create a Pull Request for this repo to add or change the details. Please following [this document](https://docs.parallel.fi/heiko-mainnet/crowdloan-v2-kusama-project-onboarding).
```

Comment:
It‘s OK for now. And once the polkadot crowdloan is open, we should change it a little bit.